src/Microsoft.AspNetCore.Sockets.Http/Internal/Transports/LongPollingTransport.cs

```diff
@@ -17,38 +17,67 @@ public class LongPollingTransport : IHttpTransport
         private readonly ILogger _logger;
         private readonly CancellationToken _timeoutToken;
         private readonly string _connectionId;
+        private readonly bool _isFirstRequest;
```

Comment:
This entire change should be reverted and we should just flush from the outside on first request and avoid calling into the long polling transport altogether.

Reply:
awesome

Reply:
Wait, so just flush a blank `200` for the first request? The change as it stands still writes the first message to the first request if there is one. The only time we have to flush a blank `200` is when there's no data in the pipe. It would make things simpler but all Long Polling connections would require two requests to initialize.

Reply:
Ah I misunderstood this change a bit. Still I hate the complexity of the control flow here. Can we expose 2 methods instead of passing the bool?

Reply:
Not sure I see how that would help, it would have to either a) Call into a single method with a bool (thus defeating the purpose) or b) Duplicate quite a bit of logic.